src/Uno.UWP/ApplicationModel/Activation/ToastNotificationActivatedEventArgs.cs

```diff
@@ -0,0 +1,8 @@
+namespace Windows.ApplicationModel.Activation
+{
+	public partial class ToastNotificationActivatedEventArgs : IToastNotificationActivatedEventArgs, IActivatedEventArgs, IActivatedEventArgsWithUser, IApplicationViewActivatedEventArgs
+	{
+		public string Argument { get; internal set; }

```

Comment:
It's generally best to use a constructor for this:

```suggestion
		internal ToastNotificationActivatedEventArgs (string argument) => Argument = arguent;
		public string Argument { get; }

```

Reply:
As you wish, although my VStudio suggests "IDE0021 Use block body for constructors" :)